lib/livebook/live_markdown/import.ex

```diff
@@ -395,7 +395,7 @@ defmodule Livebook.LiveMarkdown.Import do
       {"hub_id", hub_id}, {attrs, stamp_hub_id, messages} ->
         cond do
           Hubs.hub_exists?(hub_id) -> {Map.put(attrs, :hub_id, hub_id), hub_id, messages}
-          Hubs.get_offline_hub(hub_id) -> {attrs, hub_id, messages}
+          Hubs.get_offline_hub(hub_id) -> {Map.put(attrs, :hub_id, hub_id), hub_id, messages}
           true -> {attrs, stamp_hub_id, messages ++ ["ignoring notebook Hub with unknown id"]}
```

Comment:
I think we can revert major changes to this function. We no longer the accumulator to be `{attrs, stamp_hub_id, messages}`, it should only be `{attrs, messages}`. Then we use `attrs[:hub_id]` as the `stamp_hub_id`.